Gruntfile.js

```diff
@@ -97,11 +97,17 @@ module.exports = function(grunt) {
       tasks: 'dev'
     },
     connect: {
-      dev: {
+      preview: {
```

Comment:
`grunt connect:preview` is the old way of doing things. `grunt connect:dev` is used in conjunction with watch so you can have a server and still get new sass hotness on saves.